packages/@aws-cdk/aws-s3/lib/notifications-resource/lambda-source/run_tests.py

```diff
@@ -0,0 +1,5 @@
+import unittest
```

Comment:
You can invoke this file as part of our regular `test` command by creating a node "wrapper" test, that simply shells out to this file.

Not ideal, but good enough. 

See an example here:

https://github.com/aws/aws-cdk/blob/0952b52d037921e40b46ea975e94bf0b661c8a18/packages/%40aws-cdk/aws-s3-deployment/test/lambda.test.ts#L4-L7

Reply:
This is now working as part of the build process (takes 118 seconds though, vs 0.4 seconds when running without docker)